build.gradle

```diff
@@ -60,8 +80,6 @@ subprojects { project ->
             disableWarningsInGeneratedCode = true
             // this check is too noisy
             check("StringSplitter", CheckSeverity.OFF)
-            // https://github.com/google/error-prone/issues/3366
```

Comment:
This is not required with the latest Error Prone version (the issue was fixed), and the check did not exist in the oldest Error Prone version we support, so just delete